src/features/common/LandingVideo/styles.module.scss

```diff
@@ -117,6 +117,10 @@
     width: 15px;
   }
 }
+.embed_playButton{
+  right: 125px;
+  top: 50px;
+}
```

Comment:
These styles are for the mobile view. So the `right` value would be the same as for class`playButton`. You only need to change the value for `top`. Also, I suggest changing the value for `top` to 20px as there is a margin of 20px above the main container/embedContainer